packages/headless/src/controllers/commerce/standalone-search-box/headless-standalone-search-box-options.ts

```diff
@@ -10,6 +10,10 @@ export interface StandaloneSearchBoxOptions extends SearchBoxOptions {
    * If a query pipeline redirect is triggered, it will redirect to that Url instead.
    */
   redirectionUrl: string;
+  /**
+   * Whether to overwrite the existing standalone search box with the same id.
+   */
+  overwrite?: boolean;
```

Comment:
When would you want this to be false ? Maybe we could add more explanations in the jsdoc.

Reply:
I don't know, but I do not want to introduce a breaking change, so I added this option, because the current behavior is false always.